### DISCUSSION TITLE
Syncing failed with E022: Folder supposedly contains non-existent bookmark

### DISCUSSION THREAD

**Discussion by vatolin** [floccusaddon/floccus#1345] [Q&A] [answered]
Hi folks.

I think I did not change my system, but anyway: Today Floccus is failing to synchronize my older local bookmarks with the more recent bookmarks in my Nextcloud. Obviously there is a culprit on my bookmarks bar. The log says:

2023-02-28T18:43:35.279Z Syncing failed with E022: Folder #41353[Bookmarks Bar] supposedly contains non-existent bookmark 2634

What should I do in order to fix the failure?

How could I examine what has been caused the failure?

What could i do to prevent such failure in future?

**Comment by marcelklehr** [comment, marked as answer]
Sorry for the delay.

Are you syncing with Nextcloud Bookmarks? Did the issue resolve in the meantime? The error means that floccus received inconsistent data from the server. If you are still seeing this, you could try resetting your bookmarks on nextcloud.
